backend/backend/settings/mods/oidc.py

```diff
@@ -34,10 +34,13 @@
     OIDC["USERS"]["APPEND_DOMAIN"] = to_bool(os.environ.get("OIDC_USERS_APPEND_DOMAIN", "false"))
 
     OIDC["USERS"]["DEFAULT_CHANNEL"] = os.environ.get("OIDC_USERS_DEFAULT_CHANNEL")
-    if not OIDC["USERS"]["DEFAULT_CHANNEL"]:
-        raise Exception("No default channel provided for OIDC users")
-    if OIDC["USERS"]["DEFAULT_CHANNEL"] not in ledger.LEDGER_CHANNELS:
-        raise Exception(f"Channel {OIDC['USERS']['DEFAULT_CHANNEL']} does not exist")
+    OIDC["USERS"]["MUST_BE_APPROVED"] = to_bool(os.environ.get("OIDC_USERS_MUST_BE_APPROVED", "false"))
+    if OIDC["USERS"]["DEFAULT_CHANNEL"] and OIDC["USERS"]["MUST_BE_APPROVED"]:
+        raise Exception("Both 'default channel' and 'user must be approved' options are activated")
+    if not (OIDC["USERS"]["DEFAULT_CHANNEL"] or OIDC["USERS"]["MUST_BE_APPROVED"]):
+        raise Exception(
+            "At least one option between 'default channel' and 'user must be approved' needs to be activated"
+        )
```

Comment:
Could you add options in the chart values corresponding to this?

Right now OIDC users are given a default channel in the values here: https://github.com/Substra/substra-backend/blob/main/charts/substra-backend/values.yaml#L698

You should add a new option (for instance `oidc.users.requireApproval`), which you can link to this file here: https://github.com/Substra/substra-backend/blob/main/charts/substra-backend/templates/configmap-oidc.yaml

You'll also need to update the chart changelog and version.